spec.md

```diff
@@ -1132,13 +1126,18 @@ The result of this call will be used by CO in `ControllerPublishVolume`.
 message GetNodeIDRequest {
   // The API version assumed by the CO. This is a REQUIRED field.
   Version version = 1;
+
+  // The ID of the node as understood by the CO which MAY be returned by
```

Comment:
cc @akutz 

This is still controversial. Can we make this change a separate PR?

Reply:
Done. Will pull this in to a separate PR.